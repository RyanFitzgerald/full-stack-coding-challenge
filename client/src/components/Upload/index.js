import styled from 'styled-components';

const Upload = styled.input`
  border: 1px solid #e6e6e6;
  display: block;
  margin: 0 auto;
  padding: 5px;
`;

export default Upload;