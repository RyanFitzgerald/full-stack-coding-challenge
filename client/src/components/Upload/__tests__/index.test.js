import React from 'react';

import Upload from '../index';

describe('<Upload />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(
      <Upload type="file"/>
    ).toJSON();
    
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should render the correct element', () => {
    const renderedComponent = shallow(
      <Upload type="file"/>
    );
    
    expect(renderedComponent.type()).toEqual('input');
  });
});