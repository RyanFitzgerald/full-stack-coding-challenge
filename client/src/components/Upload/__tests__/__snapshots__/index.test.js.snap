// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Upload /> should match the snapshot 1`] = `
.c0 {
  border: 1px solid #e6e6e6;
  display: block;
  margin: 0 auto;
  padding: 5px;
}

<input
  className="c0"
  type="file"
/>
`;
